### DISCUSSION TITLE
content not move to second page

### DISCUSSION THREAD

**Discussion by atul13june** [dompdf/dompdf#3232] [Ideas]
i already updated same problem in [#3227](https://github.com/dompdf/dompdf/discussions/3227)

create a simple resume with html and css , its not move to second page with two sidebar . whenever content increase in any other sidebar then whitespace show in first page .l add a pdf file to undertand the problem
[invoice (1).pdf](https://github.com/dompdf/dompdf/files/11778370/invoice.1.pdf)

**Comment by bsweeney** [comment]
For others who run across this idea: I hope to be able to support with the updates in the paging-updates branch though I don't have a time frame for release yet.

**Comment by Bizarrus** [comment]
No code, no answer (like in #3227, #3226 and the other posts from you).
Please provide some code to check this.

Without knowing the code, my guess is the floating beginner problem.
If youre using `float` in CSS, you must also **reset the floating**, otherwise it will break the layout.

**Comment by atul13june** [comment]
you can not any reply or solution on that time so we can move to dompdf to
wkhtmltopdf ..

On Sun, Mar 17, 2024 at 9:40 AM Adrian Preuß ***@***.***>
wrote:

> No code, no answer (like in #3227
> <https://github.com/dompdf/dompdf/discussions/3227>, #3226
> <https://github.com/dompdf/dompdf/discussions/3226> and the other posts
> from you).
> Please provide some code to check this.
>
> Without knowing the code, my guess is the floating beginner problem.
> If youre using float in CSS, you must also *reset the floating*,
> otherwise it will break the layout.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dompdf/dompdf/discussions/3232#discussioncomment-8815965>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAQ3UT2B4L6X352PLTWM3GTYYUJU7AVCNFSM6AAAAAAZKC5SNGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DQMJVHE3DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thank You
Atul Sen
Web Designer
Alphawizz Technologies Pvt Ltd
